core/src/main/java/nl/nn/adapterframework/validation/SchemaUtils.java

```diff
@@ -177,10 +179,65 @@ public static Set<IXSD> mergeXsdsGroupedByNamespaceToSchemasWithoutIncludes(ISco
 		return resultXsds;
 	}
 
+	private static void addXsdMergeWarnings(final IScopeProvider scopeProvider, final Set<IXSD> xsds, final String namespace) {
+		// If there are multiple XSDs for the namespace, give ConfigWarning that explains wherefrom
+		// each file is included.
+		if (xsds.size() <= 1) {
+			return;
+		}
+		StringBuilder message = new StringBuilder("Multiple XSDs for namespace '" + namespace + "' will be merged: ");
+		for (IXSD xsd: xsds) {
+			message.append("\n - XSD path '")
+					.append(xsd.getResourceTarget())
+					.append("'");
+
+			if (xsd.getImportParent() != null) {
+				message.append(", included from '")
+						.append(xsd.getImportParent().getResourceTarget()).append("'");
+			}
+		}
+		message.append("\nPlease check that there are no overlapping definitions between these XSDs.");
+		LOG.info(message);
```

Comment:
@jkosternl @nielsm5 Vergeleken met de vorige commit is dit de functionele wijziging: ipv ConfigurationWarning toevoegen, alleen loggen.

Andere wijzigingen in de laatste commit zijn code-cleanups.